bodhi/client/__init__.py

```diff
@@ -132,6 +132,8 @@ def _warn_if_url_and_staging_set(ctx, param, value):
     click.option('--state', type=click.Choice(['disabled', 'pending', 'current',
                                                'archived']),
                  help='The state of the release'),
+    click.option('--critpath-pkgs',
+                 help='Comma-separated list of critical packages for this release.'),
```

Comment:
Looking at the current releng script to update the critpath packages, it is taking a txt file as input containing the list of packages. See (here)[https://pagure.io/releng/blob/master/f/scripts/update-critpath#_64]

So it might be a good idea to add support for this in the bodhi cli.